test/extensions/extensions_build_system.bzl

```diff
@@ -1,4 +1,4 @@
-load("//bazel:envoy_build_system.bzl", "envoy_cc_mock", "envoy_cc_test", "envoy_cc_test_library")
+load("//bazel:envoy_build_system.bzl", "envoy_cc_mock", "envoy_cc_test", "envoy_cc_test_binary", "envoy_cc_test_library")
```

Comment:
Did you mean to finish this and add an extension version?

Reply:
oh..yeah..thanks..forgot about this.